Userland/Applications/Calendar/main.cpp

```diff
@@ -100,6 +100,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     view_type_action_group->set_exclusive(true);
     view_type_action_group->add_action(*view_month_action);
     view_type_action_group->add_action(*view_year_action);
+    auto default_view = Config::read_string("Calendar"sv, "view"sv, "DefaultView"sv, "Month"sv);
```

Comment:
Sorry, I phrased that badly. I meant the group to be capitalised as well.

Reply:
Yeah, that's my bad. I should have checked how it's done elsewhere.